### PR TITLE
Add info.mumble.Mumble

### PR DESCRIPTION
**Pull request by JBBgameich**
The first Qt5 compatible release is still a release candidate, but due to the Qt4 requirement it's impossible to package the latest stable release.
https://github.com/mumble-voip/mumble/releases/tag/1.3.0-rc1

### CODE REVIEW

**Comment by LeandroStanger** [comment]
bot, build info.mumble.Mumble

**Comment by JBBgameich** [comment]
Mumble doesn't seem to include appstream metadata, will fix that.

**Comment by LeandroStanger** [comment]
bot, build info.mumble.Mumble

File: boost-configure [hunk 1]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (barthalion) [inline boost-configure]: Use `"buildsystem": "simple"` instead. Some examples: https://github.com/search?l=JSON&q=org%3Aflathub+boost&type=Code

Comment (barthalion) [reply boost-configure]: Also `$FLATPAK_BUILDER_N_JOBS` holds number of jobs to use.

File: info.mumble.Mumble.yml [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+command: mumble
+finish-args:
+  - --share=ipc
+  - --share=network
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=dri
+  - --talk-name=org.freedesktop.Notifications
+  - --own-name=info.mumble.Mumble
```
Comment (barthalion) [inline info.mumble.Mumble.yml]: Flatpak automatically grants this to app-id.

File: info.mumble.Mumble.yml [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+      - CONFIG+=c++11
+      - CONFIG+=no-g15
+      - CONFIG+=no-speechd
+      - DEFINES+=NO_UPDATE_CHECK
+    post-install:
+      - mkdir -p /app/lib/mumble/ /app/share/pixmaps/ /app/share/icons/hicolor/scalable/apps/
+        /app/share/applications/ /app/share/services/
+      - install -m 755 release/mumble /app/bin/
+      - install -m 644 release/libmumble.so* /app/lib/mumble/
+      - install -m 644 release/libcelt0.so.0.7.0 /app/lib/mumble/
```
Comment (barthalion) [inline info.mumble.Mumble.yml]: Sounds oddly specific, are you sure it won't change at some point?

File: info.mumble.Mumble.yml [hunk 4]
```diff
@@ -0,0 +1,93 @@
+id: info.mumble.Mumble
```
Comment (barthalion) [inline info.mumble.Mumble.yml]: You should add `cleanup` to remove unneeded development headers.

File: info.mumble.Mumble.appdata.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (barthalion) [inline info.mumble.Mumble.appdata.xml]: It would be preferred to get it merged upstream first to make every distribution benefit.

**Comment by barthalion** [comment]
bot, build info.mumble.Mumble

File: info.mumble.Mumble.yml [hunk 6]
```diff
@@ -0,0 +1,97 @@
...
+  - --socket=x11
+  - --socket=wayland
+  - --socket=pulseaudio
+  - --device=all
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.kde.StatusNotifierWatcher
+  - --own-name=org.kde.StatusNotifierItem-2-2
+separate-locales: false
+cleanup:
+  - /usr/include
```
Comment (barthalion) [inline info.mumble.Mumble.yml]: It should be just `/include`, flatpak-builder automatically adds `/app` prefix. Please add `/lib/pkgconfig` too.

Comment (barthalion) [reply info.mumble.Mumble.yml]: Looking at other files, does mumble need all these binaries?

```
[📦 info.mumble.Mumble app]$ ls /app/bin/
avahi-browse		   avahi-publish-address  avahi-resolve-host-name  bvnc
avahi-browse-domains	   avahi-publish-service  avahi-set-host-name	   mumble
avahi-discover-standalone  avahi-resolve	  bshell		   mumble-overlay
avahi-publish		   avahi-resolve-address  bssh			   protoc
```

File: info.mumble.Mumble.yml [hunk 7]
```diff
@@ -0,0 +1,97 @@
...
+modules:
+  - name: boost
```
Comment (barthalion) [inline info.mumble.Mumble.yml]: Looks like only headers library are required. Take a look at https://github.com/flathub/org.clementine_player.Clementine/blob/492253a161d8bee0d07eb40c4b0b781747b4a75e/org.clementine_player.Clementine.yaml#L32 how to install only these.

**Comment by barthalion** [comment]
bot, build info.mumble.Mumble

**Comment by barthalion** [comment]
bot, build info.mumble.Mumble

**Comment by barthalion** [comment]
Repository has been created: https://github.com/flathub/info.mumble.Mumble

Thanks!

**Pull request closed without merging**